File: tests/testdata/codegen/aws-secret-pp/go/aws-secret.go
package main

import (
	"github.com/khulnasoft/codeinfra-aws/sdk/v5/go/aws/rds"
	"github.com/khulnasoft/codeinfra/sdk/v3/go/codeinfra"
)

func main() {
	codeinfra.Run(func(ctx *codeinfra.Context) error {
		_, err := rds.NewCluster(ctx, "dbCluster", &rds.ClusterArgs{
			MasterPassword: codeinfra.ToSecret("foobar").(codeinfra.StringOutput),
		})
		if err != nil {
			return err
		}
		return nil
	})
}

File: tests/testdata/codegen/aws-secret-pp/dotnet/aws-secret.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using Aws = Codeinfra.Aws;

return await Deployment.RunAsync(() => 
{
    var dbCluster = new Aws.Rds.Cluster("dbCluster", new()
    {
        MasterPassword = Output.CreateSecret("foobar"),
    });

});

